purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -389,16 +416,19 @@ class PurchasesTest {
                 }
             })
 
-        assertThat(receivedProducts).isEqualTo(productDetails)
+        assertThat(receivedProducts).isEqualTo(storeProducts)
+        assertThat(receivedProducts?.size).isEqualTo(productIds.size)
     }
 
     @Test
     fun getsNonSubscriptionProducts() {
         val productIds = listOf("normal_purchase")
 
-        val storeProducts = mockStoreProduct(productIds, listOf(), ProductType.INAPP)
```

Comment:
Same issue here where empty lis to `StoreProduct`s was being returned

Reply:
that mock function is confusing to me 🥴 good find